index.html

```diff
@@ -36,13 +34,160 @@ <h2>
 don’t hestiate to contact me.
         </p>
-        </div>
+       
+       </div>
         <ul id="social-media">
-            <li><a href="#"><img src="images/github.svg" id="github"></a></li>
-            <li><a href="#"><img src="images/linkedin.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/Hand.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/M.svg" alt=""></a></li>
-        </ul>
+        <li><a href="#"><img src="images/github.svg" id="github"></a></li>
+        <li><a href="#"><img src="images/linkedin.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/Hand.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/M.svg" alt=""></a></li>
+    </ul>
+    </section>
+    <section>
+        <div id="home-2">
+        <h1 id="recent-work-heading">
+            My Recent Works</h1>
+        <img src="images/Img Placeholder.svg" id="img-placeholder">
+        
+           <h2 id="Multi-post">Multi-Post Stories</h2>
+            
+        <p id="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+         <ul id="lang-skills">
+            <li id="css">css</li>
+            <li id="html">html</li>
+            <li id="bootstrap">bootstrap</li>
+            <li id="Ruby">Ruby</li>
+       </ul>
         
-      </section>
+       <button id="project-btn">
+            See Project
+    </button>
+</div>
+    </section> 
+
+    <section id="card">
+        <div id="placehokder-img-div">
+            <div class="placehokder-img">
+        <h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
+        <p id="palceholder-img-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="lang-skills">
+            <li class="html">html</li>
+            <li class="bootstrap">bootstrap</li>
+            <li class="Ruby">Ruby</li>
+       </ul>
+       <button id="project-btn">See Project
+       </button>
+    </div>
+    </div>
+    <div id="placehokder-img-div">
+        <div class="placehokder-img">
+    <h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
+    <p id="palceholder-img-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+    <ul class="lang-skills">
+        <li class="html">html</li>
+        <li class="bootstrap">bootstrap</li>
+        <li class="Ruby">Ruby</li>
+   </ul>
+   <button id="project-btn">See Project
+   </button>
+</div>

```

Comment:
- [ ] as part of the requirements please note that the content of the cards `placehokder-img-div` should be aligned with flexbox , kindly implement it for all of your cards 🙏